🤔I wonder what's wrong here. Error on the CI:
```
/Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/branch-master/swift/test/Frontend/crash-in-user-code.swift:12:16: error: CHECK-NEXT: expected string not found in input
13:28:46 // CHECK-NEXT: While running user code "SOURCE_DIR/test/FrontEnd/crash-in-user-code.swift"
13:28:46                ^
13:28:46 <stdin>:26:1: note: scanning from here
13:28:46 3. While running user code "SOURCE_DIR/test/Frontend/crash-in-user-code.swift"
13:28:46 ^
13:28:46 <stdin>:26:4: note: possible intended match here
13:28:46 3. While running user code "SOURCE_DIR/test/Frontend/crash-in-user-code.swift"
13:28:46    ^
13:28:46 
13:28:46 --
13:28:46 
```

The test works fine on my local box. Here is an example stacktrace I am getting locally:

```
  24   │ 1.  Swift version 5.1.1-dev (Swift f2f521f8bf)
  25   │ 2.  Contents of /Users/tapanth/swift-source/build/Ninja-RelWithDebInfoAssert+swift-DebugAssert/swift-macosx-x86_64/test-macosx-x86_64/FrontEnd/Output/crash-in-user-code.swift.tmp.filelist.txt:
  26   │ ---
  27   │ /Users/tapanth/swift-source/swift/test/FrontEnd/crash-in-user-code.swift
  28   │ ---
  29   │ 3.  While running user code "/Users/tapanth/swift-source/swift/test/FrontEnd/crash-in-user-code.swift"
```

--- a/test/Frontend/crash-in-user-code.swift
+++ b/test/Frontend/crash-in-user-code.swift
@@ -9,7 +9,7 @@
 // CHECK-NEXT: ---
 // CHECK-NEXT: crash-in-user-code.swift
 // CHECK-NEXT: ---
-// CHECK-NEXT: While running user code "SOURCE_DIR/test/FrontEnd/crash-in-user-code.swift"
+// CHECK-NEXT: While running user code "SOURCE_DIR/test/Front{{e|E}}nd/crash-in-user-code.swift"
 
 let x: Int? = nil
 x!

--- a/test/Frontend/crash-in-user-code.swift
+++ b/test/Frontend/crash-in-user-code.swift
@@ -9,7 +9,7 @@
 // CHECK-NEXT: ---
 // CHECK-NEXT: crash-in-user-code.swift
 // CHECK-NEXT: ---
-// CHECK-NEXT: While running user code "SOURCE_DIR/test/Front{{e|E}}nd/crash-in-user-code.swift"
+// CHECK-NEXT: While running user code "{{.*}}crash-in-user-code.swift"
 
 let x: Int? = nil
 x!

